pyfar/classes/__init__.py

```diff
@@ -5,6 +5,7 @@
 from . import filter
 from . import transmission_matrix
 from . import warnings
+from . import _PyfarBase
```

Comment:
For file naming we usually follow the snake case convention, not capitalized camel case.

I would further suggest to collect the abstract base classes in a single file to make things a bit more compact and avoid jumping around between files when working on the base classes.

Since all base classes are not part of the public API, I would also suggest to remove the import here altogether.

``` suggestion
```

Reply:
Yes, and I would also add all future abstract base (like the multichannel, etc.) classes to the same file `_pyfar_base.py` to avoid fragmentation of related code.